audiomentations/augmentations/padding.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+
+class Padding(BaseWaveformTransform):
+    """
+    Apply padding to the audio signal -  take a fraction of the end or the start of the audio
+    and treat that part as padding.
+    """
+    
+    supports_multichannel = True
+    
+    def __init__(self, mode='constant', p=0.5):
+        super().__init__(p)
```

Comment:
Write in a docstring what values can be used for `mode`

Reply:
How about allowing used of all numpy.pad modes, not just 4 mentioned earlier?